CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -634,15 +628,24 @@ private void DestroyVertexBuffer() {
     
     // ========================> Push constants <======================== \\
     
-    private void CreatePushConstants() {
+    private int CreatePushConstants() {
         // Initialise push constants to identity mtx
-        pushConstants = memAlloc(VertexUniformBufferObject.SIZEOF);
-        for (int iRow = 0; iRow < 4; ++iRow) {
-            for (int iCol = 0; iCol < 4; ++iCol) {
-                pushConstants.putFloat(IDENTITY_44F.get(iRow, iCol));
+        try {
+            pushConstants = memAlloc(VertexUniformBufferObject.SIZEOF);
+            for (int iRow = 0; iRow < 4; ++iRow) {
+                for (int iCol = 0; iCol < 4; ++iCol) {
+                    pushConstants.putFloat(IDENTITY_44F.get(iRow, iCol));
+                }
             }
+            pushConstants.flip();
         }
-        pushConstants.flip();        
+        catch(Exception ex) {
```

Comment:
Maybe specialise this to the OOM exception if there is one?  That way if it is something else it will tank Consty and we'll see it.

Reply:
I've removed this try/catch, after googling.